docs-src/typescript/introduction-to-typescript-sdk.md

```diff
@@ -17,6 +17,8 @@ Welcome to the Temporal TypeScript SDK developer's guide!
 
 The Temporal TypeScript SDK released on July 26, 2022. The TypeScript SDK provides access to the Temporal programming model using idiomatic JavaScript and TypeScript programming paradigms.
 
+The Temporal TypeScript SDK supports the Node.js runtime environment.
```

Comment:
Can we please add "only" here, or whatever else that would clearly mean the same thing?

We regularly get questions regarding Deno, Bun, browsers… I understand the idea that we want to document what is rather than what isn't, but the goal here is to make it clear that no other runtime environment is supported at this time.

```suggestion
The Temporal TypeScript SDK only supports the Node.js runtime environment.
```